dev-lua/luacheck/luacheck-0.20.0.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit toolchain-funcs
+
+DESCRIPTION="A tool for linting and static analysis of Lua code"
+HOMEPAGE="https://github.com/mpeterv/luacheck"
+SRC_URI="https://github.com/mpeterv/luacheck/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="luajit test"
+
+RDEPEND="
+	dev-lua/luafilesystem[luajit=]
+	!luajit? ( >=dev-lang/lua-5.1 )
```

Comment:
`<` dependencies are a very bad thing™, because they prevent upgrades. Do not use them unless the thing actually does not work. Using `:=` also guarantees that the package will be rebuilt on Lua upgrade, so if anything starts failing at build time, people will notice instantly.